allauth/socialaccount/providers/authentiq/provider.py

```diff
@@ -0,0 +1,94 @@
+from allauth.account.models import EmailAddress
+from allauth.socialaccount.app_settings import \
+    QUERY_EMAIL, EMAIL_REQUIRED, EMAIL_VERIFICATION
+from allauth.socialaccount.providers.base import AuthAction, ProviderAccount
+from allauth.socialaccount.providers.oauth2.provider import OAuth2Provider
+
+
+class Scope(object):
+    NAME = 'aq:name'
+    EMAIL = 'email'
+    PHONE = 'phone'
+    EMAIL_VERIFIED = 'email~rs'
+    PHONE_VERIFIED = 'phone~rs'
```

Comment:
Actually the `_VERIFIED` constants aren't really used at all, as a developer wouldn't have easy access to them in the Django config and use the literals there. But left these in for illustration purposes; could remove too. Flags for email are managed via existing Allauth configs.